tools/arch/cockpit-wsinstance.sysuser.conf

```diff
@@ -0,0 +1 @@
+u cockpit-wsinstance - "User for cockpit-ws instances"
```

Comment:
Is there some Arch policy that they have to be separate files? The systemd-sysusers file format allows multiple lines, and these certainly belong together.

Reply:
There is no policy for those files, but I guess the maintainer found it more clear to have a separate file per user.